packages/@okta/courage-for-signin-widget/webpack.config.js

```diff
@@ -54,35 +82,28 @@ const webpackConfig = {
       'vendor/plugins/jquery.simplemodal': EMPTY,
 
       // util/BaseRouter -> ConfirmationDialog
-      'ConfirmationDialog': EMPTY,
+      '../views/components/ConfirmationDialog': EMPTY,
```

Comment:
so can `buildtools/ConfirmationDialog.ts` be deleted?

Reply:
yes